enterprise/cli/licenses.go

```diff
@@ -187,3 +192,20 @@ func (r *RootCmd) licenseDelete() *clibase.Cmd {
 	}
 	return cmd
 }
+
+func convertLicenseExpireTime(license codersdk.License) codersdk.License {
+	if license.Claims["license_expires"] != nil {
+		value, ok := license.Claims["license_expires"].(json.Number)
+		if !ok {
+			return license
+		}
+		int64Value, err := value.Int64()
+		if err != nil {
+			return license
```

Comment:
It would be clean to return errors in both conditions (here and above). I suppose that we expect only a valid number.

Reply:
ok! makes sense!